packages/ui/src/lib/dropdown/Dropdown.svelte

```diff
@@ -31,23 +35,40 @@ let {
   ariaInvalid?: boolean | 'grammar' | 'spelling';
   ariaLabel?: string;
   children?: Snippet;
+  multiple?: boolean;
+  label?: string;
 } = $props();
 
 let opened: boolean = $state(false);
 let selectLabel: string = $state('');
+let selectLabelPlus: string | undefined = $state(undefined);
 let highlightIndex: number = $state(-1);
 let comp: HTMLElement;
 
 const pageStep: number = 10;
 
 onMount(() => {
-  if (!value && options?.length > 0) {
+  if (!multiple && !value && options?.length > 0) {
     value = options[0].value;
   }
 });
 
 $effect(() => {
-  selectLabel = options.find(o => o.value === value)?.label ?? (typeof value === 'string' ? (value as string) : '');
+  if (multiple && Array.isArray(value)) {
+    selectLabelPlus = undefined;
+    if (!value.length) {
+      selectLabel = label;
+    }
+    const firstValueLabel = options.find(o => Array.isArray(value) && o.value === value[0])?.label ?? '';
+    if (value.length >= 1) {
+      selectLabel = firstValueLabel;
+    }
+    if (value.length > 1) {
+      selectLabelPlus = `+${value.length - 1}`;
+    }
+  } else {
+    selectLabel = options.find(o => o.value === value)?.label ?? (typeof value === 'string' ? (value as string) : '');
+  }
```

Comment:
This is too complex, and hard to understand

Reply:
![image](https://github.com/user-attachments/assets/74811671-c2f0-4e76-8200-68ba55145a5f)

this is not clear to the user, I would expect all values to be displayed with separated coma

Reply:
@MariaLeonova could you please give me some recommendation on how to display the selected items, when many items are selected?